playbooks/roles/wordpress/templates/default-nginx.j2

```diff
@@ -1,6 +1,15 @@
+# HTTP
 server {
 	listen *:80;
 	server_name {{ DEFAULT_SITE_HOSTNAME }};
+	return 301 https://$host$request_uri;
+}
+
+# HTTPS
+server {
+        listen 443 ssl;
+        ssl_certificate /etc/ssl/certs/{{ DEFAULT_SITE_HOSTNAME }}.crt;
```

Comment:
The certificates provisioned by certbot reside in `/etc/letsencrypt/live/DOMAIN_NAME/`, so you should change this to:
```suggestion
        ssl on;
        ssl_certificate /etc/letsencrypt/live/{{ DEFAULT_SITE_HOSTNAME }}/fullchain.pem;
        ssl_certificate_key /etc/letsencrypt/live/{{ DEFAULT_SITE_HOSTNAME }}/privkey.pem;
```

Reply:
Don't forget to remove the line below if accepting this suggestion.

Reply:
+1 to the certificates path. 

Regarding the "ssl on" directive, the ssl directive was made obsolete in Nginx 1.15.0 and Nginx doc recommends to use the ssl parameter of the listen directive  instead. Reference: http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl